boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/BoosterPacksParent.java

```diff
@@ -32,8 +33,10 @@
         for (String dep : dependencies) {
             if (dep.equals(JDBC_BOOSTER_PACK_STRING)) {
                 boosterPackConfigList.add(new JDBCBoosterPackConfigurator());
-            } else if (dep.equals(JAXRS_BOOSTER_PACK_STRING)) {
-                boosterPackConfigList.add(new JAXRSBoosterPackConfigurator());
+            } else if (dep.equals(JAXRS_BOOSTER_PACK_STRING_10) || dep.equals(JAXRS_BOOSTER_PACK_STRING_20)) {
```

Comment:
Could we just check for startsWith("io.openliberty.boosters:jaxrs:") instead? Just thinking about having to change this when new releases of the boosters come out. Would this code really need to change?

Reply:
i can do that - but I want to point out that this is the area of code that I think needs a general re-thought on design ideas. Basically was making this work for a prototype, but i don't think this approach is very efficient in the long run.